src/common/styles.js

```diff
@@ -7,4 +7,24 @@ export default StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
   },
+  card: {
+    shadowColor: '#000',
+    shadowOpacity: 0.1,
+    shadowOffset: {
+      x: 0,
+      y: 4,
+    },
+    shadowRadius: 6,
+    padding: 15,
+    margin: 15,
```

Comment:
I can accept this, but be careful. Sometimes the parent will have padding, sometimes not. I prefer when the outer space is defined by the parent. But again, if you prefer this way, just resolve the comment and I accept the pull request.